.github/workflows/nodejs.yml

```diff
@@ -9,7 +9,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [8.x, 10.x, 12.x]
+        node-version: [8.x, 10.x, 12.x, 13.x]
```

Comment:
should we ditch 8 and 10? I guess in a follow up.

Reply:
We can, but could we do when we actually see any failures?

Reply:
yeah, I'm ok with that.